examples/all-clusters-app/esp32/README.md

```diff
@@ -46,7 +46,7 @@ step. To install these components manually, follow these steps:
           $ cd ${HOME}/tools
           $ git clone https://github.com/espressif/esp-idf.git
           $ cd esp-idf
-          $ git checkout v4.3
+          $ git checkout release/v4.3
```

Comment:
@juejuegongsui We are currently using v4.3 release tag: https://github.com/espressif/esp-idf/releases/tag/v4.3 and not the branch.

Can you please provide the details about the crash you are seeing.

Reply:
Sorry, this happened when I upgraded from 4.2 to 4.3, around this Monday, I forgot to save this error

Reply:
Will it be possible for you to reproduce the crash and create an issue for the same? We can then do some analysis and decide on the next steps.

The release tag (and not the branch) is used with a purpose -- so that the CI uses the exact same IDF commit that we all use locally.